isis/src/base/objs/PvlKeyword/PvlKeyword.cpp

```diff
@@ -1528,6 +1528,15 @@ namespace Isis {
       }
     }
 
+    QRegularExpression regex("(,*[0-9]?)?");
+    QString temp = "";
+    temp += keyword;
+    QRegularExpressionMatch match = regex.match(temp);
```

Comment:
why not just pass in the string directly into the match func and not use a temp var?